contrib/docker/scripts/build.sh

```diff
@@ -21,15 +21,20 @@ export VALGRIND=${VALGRIND:-0}
 export FUZZING=${FUZZING:-0}
 export LIGHTNINGD_POSTGRES_NO_VACUUM=1
 
+pip3 install --upgrade pip
 pip3 install --user poetry
 poetry config virtualenvs.create false --local
 poetry install
 
 git clone https://github.com/lightningnetwork/lightning-rfc.git ../lightning-rfc
 git submodule update --init --recursive
 
+# Used in docker image
+cd /work || return
+ls -la
+
 ./configure CC="$CC"
-cat config.vars
+# cat config.vars
 
 cat << EOF > pytest.ini
```

Comment:
These changes seem weird?

Reply:
Yes also to me, However, for some reason, the script was run out of the work dir, and the following line just enforced to be on the right directory, and the `ls -la` is just to make the log richer, but we can remove it, but I will look where the change root directory happens

`cd /work || return`

In addition, the `cat config.vars` looks like it was not present, but it can be a refusal of my test. I will double-check that case too.

Reply:
Thanks, I found the bug in my docker config!

Let's give a run to the CI and let's see what happens!